README.md

```diff
@@ -544,91 +575,86 @@ PayCart payCart = PayCart.builder()
   .build();
 ```
 
-Additionally `PayCart` provides two functions to create and prepare 
+Additionally, `PayCart` provides two functions to create and prepare:
 
-request for Masked Wallet information:
-
-```java
-MaskedWalletRequest maskedWalletRequest = payCart.maskedWalletRequest(ANDROID_PAY_PUBLIC_KEY);
-```
+1. Request for Masked Wallet information:
+  ```java
+  MaskedWalletRequest maskedWalletRequest = payCart.maskedWalletRequest(ANDROID_PAY_PUBLIC_KEY);
+  ```
 
-request for Full Wallet inforamtion:
+2. Request for Full Wallet information:
+  ```java
+  FullWalletRequest fullWalletRequest = payCart.fullWalletRequest(maskedWallet);
+  ```
 
-```java
-FullWalletRequest fullWalletRequest = payCart.fullWalletRequest(maskedWallet);
-```
- 
- 
 ### PayHelper [⤴](#table-of-contents)
 
-Helper class that designed to simplify interaction with Android Pay. It provides next helper functions:
+`PayHelper` is a helper class that simplifies interactions with Android Pay. It provides the following helper functions:
 
 - `isAndroidPayEnabledInManifest` checks if Android Pay is enabled
 - `isReadyToPay` checks if Android Pay is ready to start purchase flow
-- `requestMaskedWallet` requests Masked Wallet information (such as billing address, shipping address, payment method etc.) from the Android Pay
+- `requestMaskedWallet` requests Masked Wallet information (such as billing address, shipping address, and payment method) from the Android Pay
 - `initializeWalletFragment` initializes wallet confirmation `SupportWalletFragment` fragment with obtained Masked Wallet information
 - `requestFullWallet` requests Full Wallet information to get payment token and complete checkout
-- `newMaskedWallet` requests Masked Wallet information from existing one with new Google Transaction Id. Usufull when user wants to retry failed purchase and current Google Transaction Id is not valid any more
-- `handleWalletResponse` helps to handle Andorid Pay wallet response by delegation callbacks via `WalletResponseHandler`
+- `newMaskedWallet` requests Masked Wallet information from existing one with new Google Transaction Id. This is useful when a user wants to retry a failed purchase and the current Google Transaction Id is no longer valid.
+- `handleWalletResponse` helps to handle Android Pay wallet response by delegation callbacks via `WalletResponseHandler`
 - and finally extract `PaymentToken` to complete the checkout, see `PayHelper#extractPaymentToken`
```

Comment:
each of the other points in this list begins with the name of a function -- is there a function that we could name here that's used to extract `PaymentToken`?